### PR TITLE
Bump heapster from v0.20.0-alpha3 to v.0.20.0-alpha4

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by fgrzadkowski** [comment]
LGTM.

File: cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml [hunk 1]
```diff
@@ -11,22 +11,22 @@ metadata:
   namespace: kube-system
```
Comment (piosz) [inline cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml]: name: heapster-v14

**Comment by piosz** [comment]
LGTM just a nit

File: cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml (hunk 1 above)
Comment (mwielgus) [inline cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml]: Done. Thanks for catching this.

**Comment by mwielgus** [comment]
The new release contains mainly bugfixes but there is always a slight chance that it may break HPA/Autoscaling/whatever. 

cc: @gmarek @lavalamp

**Comment by gmarek** [comment]
It fixes at least one panic in heapster that caused test failures, so I'm going to merge this as soon as tests are green.

**Comment by mwielgus** [comment]
Yes, it will.

**Pull request merged**